src/pretix/static/pretixcontrol/js/ui/hidequota.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+  hideDeselected();
+});
+
+function hideDeselected() {
+  var v = $("#id_quota_option").val();
+
```

Comment:
I think we can try adding in soft animation and also make the code a bit clearer and a lot shorter like the following (untested):
```
var v = $("#id_quota_option").val();
$("#existing-quota-group").slideToggle(v === "existing");
$("#new-quota-group").slideToggle(v === "new");
```
We don't even need ``hideAll`` then.